app/measure/measure.html

```diff
@@ -1,7 +1,9 @@
 <!-- Intro -->
 <section id="intro" class="wrapper style1 fullscreen">
-  <div class="featureWarning" ng-show="unsupportedSystem">
-    <p translate>Your browser may not be compatible with the M-Lab speed test. Please update to a more current version and try again.</p>
+  <div class="featureWarning" ng-show="isSafari">
+    <p translate>We're sorry, the Safari browser is not compatible with the NDT
+    test (<a href="#">more info</a>). Please use a different browser such as
```

Comment:
Could we remove this "more info" link since it doesn't currently have a destination?

Reply:
I was thinking this could be a link to a blog post about our testing and identification of the bug in Safari. We could also add this later @laiyi-ohlsen